lamindb_setup/core/upath.py

```diff
@@ -178,7 +181,26 @@ def upload_from(self, path, print_progress: bool = False, **kwargs):
             # todo: make proper progress bar for directories
             cb = fsspec.callbacks.NoOpCallback()
         kwargs["callback"] = cb
-    self.fs.upload(str(path), str(self), **kwargs)
+    # this weird thing is to avoid triggering create_bucket in upload
```

Comment:
I agree it's weird but it's fortunately only a few lines. Can you wrap it in a function?

Reply:
Hm, i also need a cleanup after upload, it would seem even worse to have some function called before upload and then some cleanup after upload.

I also hope this is a temporary solution.

Reply:
Ok!